src/github.ts

```diff
@@ -106,41 +106,48 @@ export const createTags = async (
   )
 }
 
+const generateReleaseNotes = async (
+  owner: string,
+  repo: string,
+  appOctokit: Octokit,
+  tagName: string
+): Promise<string> => {
+  try {
+    const response = await appOctokit.request(`POST /repos/${owner}/${repo}/releases/generate-notes`, {
+      owner,
+      repo,
+      tag_name: tagName,
+      target_commitish: 'main',
```

Comment:
I think this should not be main, but the current branch?
Also you do not need to set this, if you set a (already existing) tag_name